media/docs/cpp/build/building_with_sycl_support.md

```diff
@@ -69,14 +69,7 @@ $ CC=clang CXX=clang++ cmake .. -G Ninja \
 
 CMake will check that DPC++ compiler is available in the system,
 and it will download the MKL library if it cannot find it.
-To get better performance result we require the following combinations of the environment variables flags to provide better performance hints for generating optimised code. For ahead of time (AOT) compilation, the following options have to be set during compilation and for Just in time (JIT) Compilation  when running
```

Comment:
They are still needed for spir64 target. But because we don't describe that option I don't think this needs to be mentioned.